### PR TITLE
add switch to model creation method for string or integer type primar…

### PR DESCRIPTION
**Pull request by vvmruder**
…y keys

### CODE REVIEW

**Comment by vvmruder** [comment]
@sbrunner I got several issues on testing suddenly. mostly: 'E722 do not use bare except' in files i was not touching...

Was there something changed?

**Comment by vvmruder** [comment]
Addition: It is about lint errors...

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ __pycache__/
 /.cache/
 /.coverage
 /coverage_report/
+build/
```
Comment (sbrunner) [inline .gitignore]: /build/

File: pyramid_oereb/standard/create_standard_models.py [hunk 2]
```diff
@@ -39,12 +48,21 @@ def create_standard_model():
         type='string',
         help='The schema name. If not specified, it will be derived from the code.'
     )
+    parser.add_option(
+        '-s', '--primary_key_is_string',
+        dest='primary_key_is_string',
+        metavar='PRIMARY KEY IS STRING',
```
Comment (sbrunner) [inline pyramid_oereb/standard/create_standard_models.py]: This should be in one word or separate with _ with space it usually mean that we have 4 parameters...

File: pyramid_oereb/standard/create_standard_models.py [hunk 3]
```diff
@@ -9,6 +9,15 @@ def create_standard_model():
         description='Create a python file which contais all sqlalchemy models in to hold oereb '
                     'specification like data'
     )
+
+    def str2bool(v):
```
Comment (sbrunner) [inline pyramid_oereb/standard/create_standard_models.py]: never used ...

File: pyramid_oereb/standard/create_standard_models.py [hunk 4]
```diff
@@ -39,12 +48,21 @@ def create_standard_model():
...
         parser.error('No oereb code set.')
     if not options.geometry_type:
         parser.error('No geometry_type set.')
     if not options.target_path:
         parser.error('No target_path set.')
+    if options.primary_key_is_string:
+        primary_key_is_string = True
     _create_standard_configuration_models_py_(options.code, options.geometry_type, options.target_path,
-                                              options.schema)
+                                              options.schema, primary_key_is_string)
```
Comment (sbrunner) [inline pyramid_oereb/standard/create_standard_models.py]: Why having a local variable we can directly use `str2bool(options.primary_key_is_string)`

File: pyramid_oereb/standard/create_standard_models.py (hunk 3 above)
Comment (sbrunner) [reply pyramid_oereb/standard/create_standard_models.py]: => will mean that's his usage is missing :-)

Comment (vvmruder) [reply pyramid_oereb/standard/create_standard_models.py]: Ok :)) I understand :P

File: pyramid_oereb/standard/create_standard_models.py (hunk 4 above)
Comment (vvmruder) [reply pyramid_oereb/standard/create_standard_models.py]: Old leftover. I changed it.

File: .gitignore (hunk 1 above)
Comment (vvmruder) [reply .gitignore]: changed it

File: pyramid_oereb/standard/create_standard_models.py (hunk 4 above)
Comment (sbrunner) [reply pyramid_oereb/standard/create_standard_models.py]: Argparse support boolean variable, you can add `action='store_true'` in the add_option, an directly use `options.primary_key_is_string` here :-)

**Pull request merged**